### PR TITLE
PageSpeed Insights - Ensure text remains visible during webfont load

### PR DESCRIPTION
**Pull request by kanhaiya5590**
### Task
 https://pagespeed.web.dev/ - [PageSpeed Insights](https://pagespeed.web.dev/)
 
 Performance improvement - after changes result => All text remains visible during Webfont loads

Before changes - Ensure text remains visible during Webfont load
image

After changes  - All text remains visible during Webfont loads

image

### CODE REVIEW

**Comment by kanhaiya5590** [comment]
@magento run all tests

**Comment by kanhaiya5590** [comment]
**Static Tests** failed in a different place in the code - anyone can help on this as changes have not impacted that.

**Comment by paras89** [comment]
HI @kanhaiya5590, as part of the process, need to fix the static test failures in the file that was edited.

File: app/code/Magento/PageBuilder/view/frontend/web/css/source/_icons.less [hunk 1]
```diff
@@ -16,6 +16,7 @@
         src: url('@{icons-pagebuilder__font-path}.eot?#iefix') format('embedded-opentype'),
         url('@{icons-pagebuilder__font-path}.woff') format('woff'),
         url('@{icons-pagebuilder__font-path}.ttf') format('truetype');
+        font-display: swap;
```
Comment (bluemwhitew) [inline app/code/Magento/PageBuilder/view/frontend/web/css/source/_icons.less]: Nice one!

Personally, I'd use `font-display: block` in the same way `magento/theme-frontend-blank/web/css/source/_icons.less` implements. Unlike text-based fonts where you're showing _something_ before progressively "swapping" to the designated font, icons don't have anything particularly useful to show. To be clear, this still allows a period of time for swapping (_see [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display)_).

There's a [CSS Tricks](https://css-tricks.com/hey-hey-font-display) article that still stands up today that does a far better job of explaining than I could hope to (also covers the icon font use case).

In addition, I would recommend adding this to the _top_ with the other `font-*` properties for improved readability.

I don't know why this implementation doesn't use the `.lib-font-face()` wrapper, `font-style` and `font-weight` don't use `@font-style__base` and `@font-weight__regular` accordingly, etc.

**Review by bluemwhitew — CHANGES_REQUESTED** [review]

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
We are picking this PR for further development

**Comment by engcom-Hotel** [comment]
@magento run Static Tests

**Comment by engcom-Hotel** [comment]
@magento run Static Tests

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests EE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests EE, Functional Tests B2B

**Comment by engcom-Hotel** [comment]
The Failed tests seem flaky to me. Hence moving this PR in testing

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests EE, Functional Tests B2B

**Comment by engcom-Bravo** [comment]
Hi @kanhaiya5590,

Thanks for the collaboration & contribution!.

We have verified the issue in Latest 2.4-develop instance and the issue is not reproducible.Kindly refer the screenshot.

Screenshot 2024-05-27 at 12 29 32


Could you please try to reproduce the issue in Latest 2.4-develop instance and could you please let us know if we are still missing anything.

Thanks.

**Comment by engcom-Bravo** [comment]
Hi @kanhaiya5590,

We are not able to reproduce the issue.

Can you please confirm the issue still exist on 2.4-develop?

please refer to this https://github.com/magento/magento2-page-builder/pull/834#issuecomment-2133195746

Lets us know if you are still facing the issue on 2.4-develop with detailed steps.

Moving it to `on hold`

Thanks.

**Comment by engcom-Bravo** [comment]
HI @kanhaiya5590,

Thank you for your contribution!

As mentioned https://github.com/magento/magento2-page-builder/pull/834#issuecomment-2133195746, please have a look and let us know if we are missing anything.

Till then closing this PR. Please feel free to reopen it as you want to update further.

Thank you!

**Pull request closed without merging**